src/Autodiscovery/LaravelPackageManifest.php

```diff
@@ -24,7 +25,7 @@ public function collectManifestFromComposerAutoload(): Collection
         $manifest = $this->fetchManifest();
 
         if ($manifest->isEmpty()) {
-            throw new \Exception('No packages found in the manifest.');
+            throw new Exception('No packages found in the manifest.');
```

Comment:
Should we throw a specific exception here, like `AutodisoveryMismatchException`?